docs/source/guides/network_node/staking_guide.rst

```diff
@@ -238,91 +144,35 @@ Some examples:
-ask for user confirmation in the clef CLI before showing the staker accounts.
-
-To overcome this, Clef allows to define rules to automate the confirmation of certain transactions,
-or more generally, of some requests to the signer.
-In particular, we recommend that users of a Clef signer with nucypher define the following rules file,
-which simply approves the listing of accounts:
-
-.. code:: javascript
-
-    function ApproveListing() {
-        return "Approve"
-    }
-
-The sha256 digest of this particular 3-line file is ``8d089001fbb55eb8d9661b04be36ce3285ffa940e5cdf248d0071620cf02ebcd``.
-We will use this digest to attest that we trust these rules:
-
-.. code:: bash
-
-    $ clef attest 8d089001fbb55eb8d9661b04be36ce3285ffa940e5cdf248d0071620cf02ebcd
-
-    WARNING!
-
-    Clef is an account management tool. It may, like any software, contain bugs.
-
-    Please take care to
-    - backup your keystore files,
-    - verify that the keystore(s) can be opened with your password.
-
-    Clef is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
-    without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
-    PURPOSE. See the GNU General Public License for more details.
-
-    Enter 'ok' to proceed:
-    > ok
-
-    Decrypt master seed of clef
-    Password:
-    INFO [04-14|02:00:54.740] Ruleset attestation updated    sha256=8d089001fbb55eb8d9661b04be36ce3285ffa940e5cdf248d0071620cf02ebcd
-
-
-Once the rules file is attested, we can run Clef with the ``--rules rules.js`` flag,
-to indicate which are the automated rules (in our case, allowing listing of accounts):
-
-.. code:: bash
-
-    $ clef --keystore /path/to/keystore --chainid 5 --advanced --rules rules.js
-
+    $ nucypher stake create --signer clef:///home/<username>/.clef/clef.ipc --provider ~/.ethereum/geth.ipc
 
 
 Initialize a new stakeholder
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Before continuing with stake initiation and management, A setup step is required to configure nucypher for staking.
-This will create a configuration file (`~/.local/share/nucypher/stakeholder.josn`) containing editable configuration values.
+This will create a configuration file (`~/.local/share/nucypher/stakeholder.json`) containing editable configuration values.
 
 .. code:: bash
 
-    (nucypher)$ nucypher stake init-stakeholder --provider <PROVIDER> --network <NETWORK_NAME>
+    (nucypher)$ nucypher stake init-stakeholder --signer <SIGNER URI> --provider <PROVIDER> --network <NETWORK_NAME>
+
+where:
 
-If you ran ``geth`` node as above, your ``<PROVIDER>`` is
-``ipc:///home/<username>/.ethereum/goerli/geth.ipc``
-(on MacOS, ``ipc:///Users/<username>/Library/Ethereum/goerli/geth.ipc``)
+    * If you utilized :ref:`signing-with-clef`, the ``SIGNER URI`` is ``clef:///home/<username>/.clef/clef.ipc``
+      (on MacOS for Goerli testnet, ``ipc:///Users/<username>/Library/Signer/clef.ipc``)
```

Comment:
```suggestion
      (on MacOS, ``ipc:///Users/<username>/Library/Signer/clef.ipc``)
```

Reply:
@cygnusv I purposely wrote it that way because MacOS should not be used for mainnet - see https://github.com/nucypher/nucypher/pull/1945#discussion_r419611966 - I'm open if you disagree...

Reply:
But this is a _staking_ guide. MacOS users should be able to run staking commands.

Reply:
Fair point for the `clef` path, but probably not for `geth` on mainnet (L171)...

Reply:
mmm I don't see why not, although to be fair, I've never tried to run a mainnet full node in MacOS. But in principle, what's the problem with `geth` on mainnet? This is not running a node, is operating as a staker.

Reply:
✔️